test/TEST-16-EXTEND-TIMEOUT/assess.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -v -x
+
+# enough time for testsuite-success to complete
+sleep 18
```

Comment:
Just make it "big", like 90s. Various delays happen.

Reply:
test cases have been reworked a lot. The execution of this assess.sh is just waiting on a few Wants= services to start (which are largely sleep statements). I observed a timeout of ExecStartPre=/assess.sh at 90 seconds. The sentiment however is taken and 45 seconds is used to see if any services fail. I extended the other *services to take longer and have a larger margin for scheduling. Probably not to 90s level but maybe 2000s.